app/models/movie.rb

```diff
@@ -1,3 +1,14 @@
 class Movie < ApplicationRecord
   belongs_to :studio
+  has_many :movie_actors
+  has_many :actors, through: :movie_actors
+
+  def staff
+    self.actors.order(age: :desc)
+  end
+
+  def add_actor(actor)
+    self.actors << actor
```

Comment:
This looks good, but it is not strictly necessary to encapsulate this logic in a model method. This *could* live in the controller but is not wrong to put here